perf.groovy

```diff
@@ -691,6 +691,109 @@ parallel(
     }
 }
 
+// Setup size-on-disk test
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+        ['x64', 'x86'].each { arch ->
+            def architecture = arch
+            def newJob = job(Utilities.getFullJobName(project, "sizeondisk_${arch}", isPR)) {
+
+                // Set the label.
+                label('20170427-elevated')
```

Comment:
Instead of doing this you should use the setMachineAffinity function.  You can find it's use in the smoketest parts of the other perf jobs.  You should use this as they update the actual Jenkins label often,and update that function to keep returning the correct thing.